test/kb_blast_server_test.py

```diff
@@ -71,33 +71,156 @@ def getImpl(self):
     def getContext(self):
         return self.__class__.ctx
 
+    # get obj_ref in form D/D/D
+    def get_obj_ref_from_obj_info (self, obj_info):
+        [OBJID_I, NAME_I, TYPE_I, SAVE_DATE_I, VERSION_I, SAVED_BY_I, WSID_I, WORKSPACE_I, CHSUM_I, SIZE_I, META_I] = list(range(11))  # object_info tuple
+
+        return '/'.join([str(obj_info[WSID_I]), str(obj_info[OBJID_I]), str(obj_info[VERSION_I])])
+    
+
+    # call this method to get the WS object info of a Genome
+    #   (will upload the example data if this is the first time the method is called during tests)
+    def getGenomeInfo(self, genome_basename, item_i=0):
+        if hasattr(self.__class__, 'genomeInfo_list'):
+            try:
+                info = self.__class__.genomeInfo_list[item_i]
+                name = self.__class__.genomeName_list[item_i]
+                if info != None:
+                    if name != genome_basename:
+                        self.__class__.genomeInfo_list[item_i] = None
+                        self.__class__.genomeName_list[item_i] = None
+                    else:
+                        return info
+            except:
+                pass
+
+        # 1) transform genbank to kbase genome object and upload to ws
+        shared_dir = "/kb/module/work/tmp"
+        genome_data_file = 'data/genomes/'+genome_basename+'.gbff.gz'
+        genome_file = os.path.join(shared_dir, os.path.basename(genome_data_file))
+        shutil.copy(genome_data_file, genome_file)
+
+        SERVICE_VER = 'release'
+        #SERVICE_VER = 'dev'
+        GFU = GenomeFileUtil(os.environ['SDK_CALLBACK_URL'],
+                             token=self.getContext()['token'],
+                             service_ver=SERVICE_VER
+                         )
+        print ("UPLOADING genome: "+genome_basename+" to WORKSPACE "+self.getWsName()+" ...")
+        genome_upload_result = GFU.genbank_to_genome({'file': {'path': genome_file },
+                                                      'workspace_name': self.getWsName(),
+                                                      'genome_name': genome_basename
+                                                  })
```

Comment:
It would be beneficial to run a code formatter over this codebase to make it easy for anyone coming in to read it. We use `black` in repos like the narrative and the relation engine, and it ensures everything is formatted in a standardised way, uniform use of indentation, braces, quotation marks, etc.

Reply:
will do with a PR for just that.  Will confuse this PR if formatter has changed everything.